deblur/workflow.py

```diff
@@ -131,34 +152,59 @@ def remove_artifacts_seqs(seqs_fp,
         number of threads to use for SortMeRNA
     verbose: boolean, optional
         If true, output SortMeRNA errors
+    sim_thresh: float, optional
```

Comment:
It will probably also be good to mention that the value should be between 0 and 1.